{
  "id": "CVE-2006-0105",
  "package_name": "postgresql",
  "fix_state": "unknown",
  "cpes": [
    "cpe:2.3:a:postgresql:postgresql:8.1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.0.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:postgresql:postgresql:8.1.0:*:*:*:*:*:*:*"
  ],
  "version_constraint": "= 8.0 || = 8.0.1 || = 8.0.2 || = 8.0.3 || = 8.0.4 || = 8.0.5 || = 8.1.0 || = 8.1.1",
  "fixed_in_versions": "null",
  "severity": "Medium",
  "description": "PostgreSQL 8.0.x before 8.0.6 and 8.1.x before 8.1.2, when running on Windows, allows remote attackers to cause a denial of service (postmaster exit and no new connections) via a large number of simultaneous connection requests.",
  "urls": [
    "http://archives.postgresql.org/pgsql-announce/2006-01/msg00001.php",
    "http://securityreason.com/securityalert/327",
    "http://securitytracker.com/id?1015482",
    "http://www.postgresql.org/about/news.456",
    "http://www.securityfocus.com/archive/1/421592/100/0/threaded",
    "http://www.securityfocus.com/bid/16201",
    "http://www.vupen.com/english/advisories/2006/0114",
    "https://exchange.xforce.ibmcloud.com/vulnerabilities/24049"
  ]
}
